executor_manager/main.py

```
@@ -81,18 +81,52 @@ async def lifespan(app):
+            # Online consumer - processes tasks immediately
+            task_consumer = TaskQueueConsumer(service_pool, queue_type="online")
+            task_consumer.start()
+            logger.info(f"Online task queue consumer started for pool '{service_pool}'")
+
+            # Offline consumer - only processes during 21:00-08:00
+            offline_consumer = TaskQueueConsumer(service_pool, queue_type="offline")
+            offline_consumer.start()
+            logger.info(
+                f"Offline task queue consumer started for pool '{service_pool}'"
+            )
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)